### ISSUE TITLE
Unexpected String Rendering

### ISSUE THREAD

**Issue by smujmaiku** [tulios/json-viewer#78]
When only a quoted string is parsed some strange errors occur.  Looks like some characters cause new lines and spaces are removed:

Raw View:
```
"## Strings\nThis here, is a string"
```

Parsed View:
```
// 20160415172152
// https://smujmaiku.com/test.json

"##Strings\nThishere,
isastring"
```

**Comment by tulios** [comment]
Interesting bug, adding an image to make the bug clear.

![screen shot 2016-04-21 at 11 11 18](https://cloud.githubusercontent.com/assets/33231/14703740/d1de90f0-07b1-11e6-8a00-fd81634239e0.png)

Thanks for reporting.

**Comment by tulios** [comment]
I think this is related to issue #77

**Comment by tulios** [comment]
This error is fixed in version 0.14.0, thanks for the reporting.
